src/main/java/uc/user/login/LoginInputBoundary.java

```diff
@@ -2,12 +2,13 @@
 
 /**
  * LoginInputBoundary provides an entry-point to the login use case.
- * Used by a Controller and implemented by LoginInteractor.
+ * Used by LoginController and implemented by LoginInteractor.
```

Comment:
Not sure if this is good idea, since documentation for use cases should not know anything about the layer above (I was reluctant to even refer to a controller).